s3/pkg/meta/db/drivers/tidb/bucket.go

```diff
@@ -113,17 +113,19 @@ func (t *TidbClient) GetBucket(ctx context.Context, bucketName string) (bucket *
 	}
 
 	// get SSE info for this bucket
+	tmp.ServerSideEncryption = &pb.ServerSideEncryption{}
 	sseOpts, sseErr := t.GetBucketSSE(ctx, tmp.Name)
 	if sseErr != nil {
 		return
 	}
 	sseType := "NONE"
 	if sseOpts != nil {
 		sseType = sseOpts.SseType
+		tmp.ServerSideEncryption.EncryptionKey = sseOpts.EncryptionKey
+		log.Infoln("encryption key is")
```

Comment:
These two can be merged into one line

Reply:
sheesh !!! missed removing debug logs, these are removed. thanks